playwright.config.ts

```diff
@@ -0,0 +1,80 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// import dotenv from 'dotenv';
```

Comment:
does playwright do something with this or is it just commented out code?

Reply:
it is an autogenerated file where this was commented by default. we can uncomment once we start using env files